notebooks/community/ml_ops/stage3/get_started_vertex_pipelines_xgboost_with_prediction.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>The arguments dataset_data_url and dataset_labels_url in the markdown are not consistent with the ones defined in the code below.</p><p> </p><p>parser.add_argument("--dataset-data-url", dest="dataset_data_url",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;type=str, help="Download url for the training data.")</p><p>parser.add_argument("--dataset-labels-url", dest="dataset_labels_url",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;type=str, help="Download url for the training data labels.")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Ah, the - vs _ again. Done